packages/gestalt-design-tokens/tokens/vr-theme/base-line-height.json

```diff
@@ -0,0 +1,30 @@
+{
+  "font": {
+    "line-height": {
```

Comment:
How do the "-" translates to token?
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/4392bc2e-4902-4bcc-b993-4b3c04181fb7)

It creates a deeper level no?

Reply:
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/fde10f84-aca6-42a0-bfdd-e20ea6b7932c)



Should we just do lineHeight? we decided something similar with data-visualization, we went for dataviz instead 
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/9b7d6279-67ec-40d4-b883-5c9708b7482f)

Reply:
I'm okay with `line-height`. They're two separate words. I thought for data-viz one of the other issues was the sema and base naming conflict

This was the example from Android I saw, so I think it's expected:

<img width="316" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/eb85853a-d2bc-4204-a569-65a3d941f289">